web/scripts/load_agagd_data.py

```diff
@@ -52,20 +52,27 @@ def get_or_make_user(self, aga_id):
         be used.
         """
         while aga_id in self._pin_changes:
+            if self._pin_changes[aga_id] is None:
+                print ("Pin would change to none: %s" % aga_id)
```

Comment:
The data was actually ok w.r.t. these Nones.  i had a bug somewhere else and tracked it down here.  Reverted